### DISCUSSION TITLE
Welcome to APT Discussions!

### DISCUSSION THREAD

**Discussion by kristinbranson** [kristinbranson/APT#370] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by roian** [comment]
I'm using APT to track mice and their bodies are very flexible.

**Comment by pascar4** [comment]
Just curious, is there an easy way to transfer labeled frames from one project to another?

**Reply by arobie** [reply]
I think the best way to do this is by exporting labels and then importing them into another project. File ->  Import/Export -> Advanced -> Export labels as table 
I recommend table because then they can all be imported at once rather than per movie. 
You do need to have the movies added to project before you import labels for them.
